src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -1,10 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>Microsoft.Spark</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
+
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <Description>.NET for Apache Spark</Description>
+    <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+    <PackageProjectUrl>https://github.com/dotnet/spark</PackageProjectUrl>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageTags>spark;dotnet;csharp</PackageTags>
+
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
Now we are creating `.snupkg` packages, but we aren't doing anything with them in the build. Can we publish them somewhere? Like in a "BuildArtifact"?

Reply:
Ideally we would publish them to the Microsoft symbol server, but that is probably outside of the scope of this PR. We can do that in the future.

---
In reply to: [278755885](https://github.com/dotnet/spark/pull/54#discussion_r278755885) [](ancestors = 278755885)

Reply:
I added a step to publish them as artifacts. Does it help?